### DISCUSSION TITLE
Cannot compile C#  project

### DISCUSSION THREAD

**Discussion by yibo7** [CoderLine/alphaTab#628] [Q&A] [answered]
Who can help me
Open alphatab.sln and compile
AlphaTab.Midi,AlphaTab.Synth,AlphaTab.Model not found

**Comment by Danielku15** [comment, marked as answer]
You will need to follow the [contribution guide here](https://alphatab.net/docs/contributing#join-the-code) to understand the setup of our toolchain and how you build alphaTab.

If you just want to use alphaTab you might want to rather check out the [Getting Started Guide for .net](https://alphatab.net/docs/getting-started/installation-net) or the [Tutorial for .net](https://alphatab.net/docs/tutorial-net/introduction)

**Comment by yibo7** [comment]
Thank you very much. I'm learning how to build alphaTab.
